packages/scss/src/components/inputs/checkbox/_checkbox.scss

```diff
@@ -11,42 +11,33 @@
 		&::after {
 			transform: scale(1);
 		}
+		&::before {
+			@include makeIcon("tick");
+		}
 	}
 }
 
 .checkbox-label {
 	cursor: pointer;
-	display: inline-block;
-	padding: _component("checkbox.label.padding");
-	line-height: _component("checkbox.label.line-height");
+	display: flex;
```

Comment:
checkbox-label should not but full width elements as it can lead to unwanted clicks on label.